README.md

```diff
@@ -21,29 +32,52 @@ import (
 )
 
 func main {
-    data, _ := os.ReadFile("./test/data/invoice-de-de.json")
+    data, err := os.ReadFile("./test/data/invoice-de-de.json")
+    if err != nil {
+        panic(err)
+    }
 
     env := new(gobl.Envelope)
     if err := json.Unmarshal(data, env); err != nil {
         panic(err)
     }
 
-    // Prepare the CFDI document
-    doc, err := xinvoice.NewDocument(env)
+    doc, err := xinvoice.ConvertToXRechnungCII(env)
```

Comment:
I think it would be useful in the example to show what you should do with the output, if its byte array or an object.